scripts/jobs/validate/test

```diff
@@ -1,17 +1,46 @@
-#!/bin/bash -e
+#!/bin/bash -e -v -x
+
+baseDir=${WORKSPACE-`pwd`}
+scriptsDir="$baseDir/scripts"
+. $scriptsDir/common
+
+function runsbt() {
+  $SBT_CMD \
+     -Dstarr.version=$scalaVersion \
```

Comment:
We should introduce an sbt SettingKey for starr.version. Once ant is gone we can remove the old versions.properties stuff but the key could already be used now (so you'd call an sbt set command here instead of overriding it through a system property).